language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/position/PositionTreeVisitor.java

```diff
@@ -141,6 +141,20 @@ public void visit(BLangFunction funcNode) {
             return;
         }
 
+        if (HoverUtil.isMatchingPosition(HoverUtil.getIdentifierPosition(funcNode), this.position)) {
```

Comment:
Is there any possibility to introduce a common method for the following duplications?

Reply:
+1, will move to a common method